Resources/Prototypes/Nyanotrasen/Recipes/Construction/Graphs/furniture/tables.yml

```diff
@@ -0,0 +1,45 @@
+- type: constructionGraph
+  id: Table
```

Comment:
I believe this may conflict with the other Table constructionGraph, but I would have to check. A simple copy and paste may not work here.